spartan/aztec-network/values.yaml

```diff
@@ -25,7 +25,7 @@ bootNode:
   externalUdpHost: ""
   replicas: 1
   service:
-    p2pTcpPort: 40400
+    p2pTcpPort: 40500
```

Comment:
hmm what's the issue with the same port being used?

Reply:
None, this change was when I was messing about with routing UDP into the cluster. Will revert